apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Tests/GoogleStackdriverTargetTest.cs

```diff
@@ -134,16 +148,128 @@ private LogEventInfo CreateLog(LogLevel level, string msg)
                 CredentialFile = Layout.FromString("not_empty"),
                 CredentialJson = Layout.FromString("not_empty"),
             };
-            // ConfigureForTargetLogging() and a write are both required to trigger target initialization.
-            SimpleConfigurator.ConfigureForTargetLogging(target);
-            var tcs = new TaskCompletionSource<int>();
-            target.WriteAsyncLogEvent(new AsyncLogEventInfo(LogEventInfo.CreateNullEvent(), MakeContinuation(tcs)));
-            var nlogEx = await Assert.ThrowsAsync<NLogRuntimeException>(() => tcs.Task);
+            var nlogEx = await Assert.ThrowsAsync<NLogRuntimeException>(() => ActivateTargetAsync(target));
             var innerEx = nlogEx.InnerException;
             Assert.IsType<InvalidOperationException>(innerEx);
             Assert.True(innerEx.Message.Contains("CredentialFile") && innerEx.Message.Contains("CredentialJson"));
         }
 
+        [Fact]
+        public async Task InvalidOptions_MultipleJsonConverters()
+        {
+            var target = new GoogleStackdriverTarget
+            {
+                ProjectId = "a_project_id",
+                SendJsonPayload = true,
+                JsonConvertFunction = o => Value.ForNull(),
+                JsonConvertTypeName = "a_type_name",
+            };
+            var nlogEx = await Assert.ThrowsAsync<NLogRuntimeException>(() => ActivateTargetAsync(target));
+            Assert.IsType<InvalidOperationException>(nlogEx.InnerException);
+        }
+
+        [Fact]
+        public async Task InvalidOptions_BadJsonConverterType()
+        {
+            var target = new GoogleStackdriverTarget
+            {
+                ProjectId = "a_project_id",
+                SendJsonPayload = true,
+                JsonConvertTypeName = "a_type_name",
+                JsonConvertMethodName = "a_method_name",
+            };
+            var nlogEx = await Assert.ThrowsAsync<NLogRuntimeException>(() => ActivateTargetAsync(target));
+            Assert.IsType<InvalidOperationException>(nlogEx.InnerException);
+            Assert.Contains("a_type_name", nlogEx.InnerException.Message);
+        }
+
+        private const string ConverterMsg = "Converter called :)";

```

Comment:
Any reason this is here, between two InvalidOptions tests?

Reply:
It's because the InvalidOption test beneath this class definition uses the class.
Mind you, the const string isn't needed any more, so got rid of that.